content/updates/2016-12-14-release-notes.md

```diff
@@ -0,0 +1,36 @@
++++
+date = "2016-12-14"
+title = "Platform Release Notes"
++++
+
+Curious what’s new that you might find helpful as a cloud.gov application developer? Here are highlights from our platform updates over the past two weeks.
+<!--more-->
+
+### Added
+- New [dashboard](https://dashboard.fr.cloud.gov) features
+  - The number of application instances, instance memory and disk quota can now be edited from the application page.
+  - Org and space level views now include a quick overview of the health of application instances.
+- We’ve launched [cloud.gov community](https://community.cloud.gov), a new resource for asking questions, and sharing best practices for using cloud.gov.
+
+### Changed
+- We upgraded Cloud Foundry (the underlying open source project that powers cloud.gov) to [Release v249](https://github.com/cloudfoundry/cf-release/releases/tag/v249) (see also [Release v248](https://github.com/cloudfoundry/cf-release/releases/tag/v248)), which includes updates to buildpacks and improvements for internal components.
+- CloudFoundry upgraded the command line interface (CLI) client. Please upgrade to the [latest version](https://github.com/cloudfoundry/cli/releases/latest).
```

Comment:
I'd be specific here... "Changes in Cloud Foundry require an upgrade to the latest version of the CLI in order to `cf logs` to work properly.